examples/src/test/java/ai/djl/examples/training/TrainAirfoilWithTabNetTest.java

```diff
@@ -25,6 +25,7 @@ public class TrainAirfoilWithTabNetTest {
 
     @Test
     public void testTrainAirfoilWithTabNet() throws TranslateException, IOException {
+        TestRequirements.nightly();
```

Comment:
We already reduced continuous build epoch, should be OK for continuous build now